platform/packaging/build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM docker.io/library/debian:12
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Phase 1: Install free components.
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+    python3-dev python3-numpy-dev swig python3-virtualenv \
+    cmake git gpg wget
+
+# Phase 2: Install non-free components.
+RUN sed -i -e 's/main/main contrib non-free/g' /etc/apt/sources.list.d/debian.sources
+
+# Add nVidia CUDA repository.
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/debian11/x86_64/cuda-keyring_1.1-1_all.deb
```

Comment:
For future, please use `ADD` instead of `RUN wget`.

It has many beneficial properties that an in-image `RUN` lacks.

Thank you,.